service/backend/pchain/indexer/parser.go

```diff
@@ -47,78 +42,59 @@ type Parser interface {
 var _ Parser = &parser{}
 
 type parser struct {
-	networkID   uint32
-	avaxAssetID ids.ID
-	aliaser     ids.Aliaser
+	networkID uint32
+	aliaser   ids.Aliaser
 
 	codec        codec.Manager
 	codecVersion uint16
 
 	ctx *snow.Context
```

Comment:
I am not really sure what is the purpose of this ctx? 
Can't we use the one provided in every method of Parser interface?